src/rpdk/core/data/schema/base.definition.schema.v1.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Can we not add this manually here, instead update https://github.com/aws-cloudformation/cloudformation-cli/blob/master/.github/workflows/schema-updater.yaml. Same with other schema definitions too, we should include in the above action to automatically pull from the resource-schema package.

Reply:
+1, this reverted commits like @Yunhao-Jiao's https://github.com/aws-cloudformation/cloudformation-resource-schema/pull/123